packages/core/src/components/ChoiceList/Choice.scss

```diff
@@ -195,6 +195,8 @@ $ds-c-inset-border-width: $spacer-half;
 /*
 Right-to-Left
 
+<span class="ds-c-badge ds-u-margin-right--1 ds-u-text-transform--capitalize">Deprecated</span>
```

Comment:
@status Deprecated doesn't work here I believe.

Reply:
You are right it only works in the KSS flags portion of the documentation